parquet-scala/pom.xml

```diff
@@ -46,13 +46,13 @@
       <url>https://scala-tools.org/repo-releases</url>
     </pluginRepository>
   </pluginRepositories>
-    
+
```

Comment:
Probably IDE does that but let's avoid unnecessary change here. If other pending PRs touch the same line, it would conflict.

Reply:
@shangxinli I've re-introduced the empty spaces!